scripts/manager.mjs

```diff
@@ -366,38 +368,8 @@ export default class ChexManager {
     }
 
     getHexFromPoint(point) {
-        const grid = canvas.grid.grid;
-		const [row, col] = canvas.grid.grid.getGridPositionFromPixels(point.x, point.y);
-		return this.hexes.get(ChexData.getKey({row, col}));
+		const {i, j} = canvas.grid.getOffset(point);
+		return this.hexes.get(ChexData.getKey({i, j}));
```

Comment:
nit: Just realizing I could just get the return value of the getOffset here and pass it in directly. This is what happens when you code at 1 AM. If there are other changes asked for, I'll make sure to clean this up.

Reply:
Heh, I know the feeling. No worries, I'll do a quick review before I merge it. With a bit of luck I still understand my own code.